### PR TITLE
Update g-suite-apps-audit-source.md

### PR DESCRIPTION
**Pull request by kparekh010**
## Purpose of this pull request

This pull request (PR) is to add third party app access information to allow access to gsuite data. 

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source.md [hunk 1]
```diff
@@ -37,6 +37,15 @@ When creating or modifying a Google Apps Audit Source, you will be required to a
...
 :::
 
+### Manage Third party app access
+In order to provide Sumo logic access to google services, please follow the manage third-party app access steps as follows. See Google's [Manage Third party App Access](https://support.google.com/a/answer/7281227#zippy=%2Cstep-manage-third-party-app-access-to-google-services-add-apps) documentation for more details. 
+1. In the Admin console, go to Menu —> Security -> Access and data control -> API controls.
+1. In App access control, click Manage Third-Party App Access.
+1. Click “Add app” and search for sumo logic o-auth client.
+1. Select sumo o-auth client and click change access. 
+1. Choose the trusted option to allow access to all google services. 
+1. Click change.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source.md:47]:
```suggestion
In order to provide Sumo logic access to google services, please follow the manage third-party app access steps as follows. See Google's [Manage Third-party App Access](https://support.google.com/a/answer/7281227#zippy=%2Cstep-manage-third-party-app-access-to-google-services-add-apps) documentation for more details. 
1. In the Admin console, go to **Menu** > **Security** > **Access and data control** > **API controls**.
1. In App access control, click **Manage Third-Party App Access**.
1. Click **Add app** and search for Sumo Logic o-auth client.
1. Select Sumo Logic o-auth client and click **Change access**. 
1. Choose the trusted option to allow access to all Google services. 
1. Click **Change**. 
```

File: docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source.md [hunk 2]
```diff
@@ -37,6 +37,15 @@ When creating or modifying a Google Apps Audit Source, you will be required to a
 Authentication must be with a new G Suite Apps Audit Source, we do not support re-authenticating existing sources.
 :::
 
+### Manage Third party app access
+In order to provide Sumo logic access to google services, please follow the manage third-party app access steps as follows. See Google's [Manage Third-party App Access](https://support.google.com/a/answer/7281227#zippy=%2Cstep-manage-third-party-app-access-to-google-services-add-apps) documentation for more details. 
+1. In the Admin console, go to **Menu** > **Security** > **Access and data control** > **API controls**.
+1. In App access control, click **Manage Third-Party App Access**.
+1. Click **Add app** and search for Sumo Logic o-auth client.
+1. Select Sumo Logic o-auth client and click **Change access**.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source.md:45]: @kparekh010 , can you confirm the exact wording (and capitalization) in the UI? Is it **Sumo Logic O-Auth client**?

File: docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source.md [hunk 3]
```diff
@@ -37,6 +37,15 @@ When creating or modifying a Google Apps Audit Source, you will be required to a
 Authentication must be with a new G Suite Apps Audit Source, we do not support re-authenticating existing sources.
 :::
 
+### Manage Third party app access
+In order to provide Sumo logic access to google services, please follow the manage third-party app access steps as follows. See Google's [Manage Third-party App Access](https://support.google.com/a/answer/7281227#zippy=%2Cstep-manage-third-party-app-access-to-google-services-add-apps) documentation for more details. 
+1. In the Admin console, go to **Menu** > **Security** > **Access and data control** > **API controls**.
+1. In App access control, click **Manage Third-Party App Access**.
+1. Click **Add app** and search for Sumo Logic o-auth client.
+1. Select Sumo Logic o-auth client and click **Change access**. 
+1. Choose the trusted option to allow access to all Google services.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source.md:46]: @kparekh010 is this a dropdown in the UI?

**Review by kimsauce — CHANGES_REQUESTED** [review]
Style Guide edits and formatting Qs

File: docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source.md (hunk 2 above)
Comment (kparekh010) [reply docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source.md:45]: gsuite-sumo-third-party-app
Hi Kim, You can see the sumo logic web app in this image, this is the one customers need to add and give access to. I received this image from a current prospect.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**